plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -386,6 +386,87 @@ public void testShowCreateTable()
                         ")");
     }
 
+    @Test
+    public void testNumericType()
```

Comment:
I would remove this test case if `TestBigQueryTypeMapping.java` is acceptable.

Reply:
I like the type mapping tests you've added. And I think we want to use TypeMapping tests eventually.